packages/react-scripts/scripts/utils/frontierInit.js

```diff
@@ -48,6 +53,9 @@ function installFrontierDependencies(appPath, answers, useYarn, ownPath) {
   if (additionalFeatures.includes('electric-flow')) {
     configureEF(appPath, useYarn, ownPath);
   }
+  if (additionalFeatures.includes('header-footer')) {
+    configureHF(appPath, useYarn, ownPath);
```

Comment:
not specific to hf, but I don't think we should allow yarn as an option

Reply:
I'm fine with that I guess, but if a developer doesn't use the --use-npm option, then they'll get react, react-dom, and @fs/react-scripts installed with yarn, and then all of "our" dependencies installed with npm. 

I don't know if anyone would ever use create-react-app that way though, they should just use the fr-cli.

Reply:
I'll go ahead and make this change to remove the yarn option completely in our frontierInit code